core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -266,7 +266,28 @@
   public static final String IDLE_TIMEOUT_MS_CONFIG = "idle.timeout.ms";
   public static final String IDLE_TIMEOUT_MS_DOC =
           "The number of milliseconds to hold an idle session open for.";
-  public static final long IDLE_TIMEOUT_MS_DEFAULT = 30_000;
+  public static final long IDLE_TIMEOUT_MS_DEFAULT = 60_000;
```

Comment:
why is the idle timeout default changing?

Reply:
The default value for idle timeout is 60_000 in Jetty. I do not know why it was 30_000 in rest-utils. So I changed it back to 60_000.

Reply:
Changed back to 30_000 due I think this might break other REST application.